UM/Qt/qml/UM/Preferences/MachinesPage.qml

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015 Ultimaker B.V.
+// Uranium is released under the terms of the AGPLv3 or higher.
+
+import QtQuick 2.1
+import QtQuick.Controls 1.1
+
+import UM 1.1 as UM
+
+ManagementPage {
```

Comment:
We should put { on newline. But let's fix that in a general QML style cleanup once we establish what that coding style should be.